arbitrum-docs/partials/_contribute-core-docs-partial.md

```diff
@@ -0,0 +1,13 @@
+1.  Begin by creating a branch of the [Arbitrum docs repo](https://github.com/OffchainLabs/arbitrum-docs).
+2.  Issue a `Draft` pull request from your branch into `master`. Pull requests into `master` generate a preview of your changes via a branch-specific Docusaurus deployment; this preview will update as you push commits to your remote branch.
+3.  Include answers to the following questions in your PR description:
+    ```markdown
+    1. Audience: Who am I writing for?
+    2. Problem: What specific problem are they trying to solve?
+    3. Discovery: How are they looking for a solution to this problem? What search terms are they using?
+    4. Document type: Which document type is most suitable?
+    5. Validated demand: How do we know that this documentation is actually needed?
```

Comment:
I feel like this bar could be kinda tricky to meet. Maybe some examples of how demand could be validated by 3p contributors would be useful

Reply:
Yeah... I think items (1) - (3) more concretely address the objective here (making sure we're targeting a specific need)... will cut (5)